packages/nodes-base/nodes/Oracle/Sql/actions/common.descriptions.ts

```diff
@@ -0,0 +1,547 @@
+import type { INodeProperties } from 'n8n-workflow';
+
+const stmtBatchOptions = [
+	{
+		name: 'Single Statement',
+		value: 'single',
+		description: 'A single Statement for all incoming items',
+	},
+	{
+		name: 'Independently',
+		value: 'independently',
+		description: 'Execute one Statement per incoming item of the run',
+	},
+	{
+		name: 'Transaction',
+		value: 'transaction',
+		description:
+			'Execute all Statements in a transaction, if a failure occurs, all changes are rolled back',
+	},
+];
+
+export const optionsCollection: INodeProperties[] = [
+	{
+		displayName: 'Options',
+		name: 'options',
+		type: 'collection',
+		placeholder: 'Add option',
+		default: {},
+		options: [
+			{
+				displayName: 'Auto Commit',
+				name: 'autoCommit',
+				type: 'boolean',
+				default: true,
+				description:
+					'Whether this property is true, then the transaction in the current connection is automatically committed at the end of statement execution',
+			},
+			{
+				displayName: 'Bind Variable Placeholder Values',
+				name: 'params',
+				placeholder: 'Add Parameter',
+				type: 'fixedCollection',
+				typeOptions: {
+					multipleValueButtonText: 'Add another Parameter',
+					multipleValues: true,
+				},
+				displayOptions: {
+					show: {
+						'/operation': ['execute'],
+					},
+				},
+				default: {},
+				description: 'Enter the values for the bind parameters used in the statement',
+				options: [
+					{
+						displayName: 'Values',
+						name: 'values',
+						values: [
+							{
+								displayName: 'Bind Name or Number',
+								name: 'name',
+								type: 'string',
+								default: '',
+								description: 'A bind variable placeholder identifier or numeral',
+								placeholder:
+									'e.g. dept_id`` and ``dept_name`` are the two bind variables placeholders in this SQL statement',
```

Comment:
I think it should be either:
```suggestion
									'e.g. ``dept_id`` and ``dept_name`` are the two bind variables placeholders in this SQL statement',
```
or:
```suggestion
									'e.g. `dept_id` and `dept_name` are the two bind variables placeholders in this SQL statement',
```